### PR TITLE
chore(RN): add release notes for migration v 2.46.0

### PR DESCRIPTION
**Pull request by baptistemesta**

### CODE REVIEW

File: md/migration-tool.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ to the one delivered with the latest version of Bonita.
 This is due to the fact that improvements in any version of the Migration Tool can affect all supported versions of Bonita.
 This changelog must be read before [migrating to a newer version of Bonita](migrate-from-an-earlier-version-of-bonita-bpm.md).
 
+## 2.46.0 - XXXX YYYY, 2019 (Bonita 7.10.0)
+This version of the migration tool migrates Bonita up to version 7.9.4.
```
Comment (educhastenier) [inline md/migration-tool.md]:
```suggestion
This version of the migration tool migrates Bonita up to version 7.10.0.
```

File: md/migration-tool.md [hunk 2]
```diff
@@ -5,6 +5,14 @@ to the one delivered with the latest version of Bonita.
...
+* Add a live-migration tool to reintgrate arch_contract_data_back into arch_contract_data table
```
Comment (educhastenier) [inline md/migration-tool.md]:
```suggestion
* Add a live-migration tool to reintegrate arch_contract_data_backup into arch_contract_data table
```

File: md/migration-tool.md [hunk 3]
```diff
@@ -5,6 +5,14 @@ to the one delivered with the latest version of Bonita.
...
+
+## 2.45.0 - October 3rd, 2019 (Bonita 7.9.4)
+This version of the migration tool migrates Bonita up to version 7.9.4.
+* Add a new script `check-migration-dryrun`
```
Comment (educhastenier) [inline md/migration-tool.md:14]:
```suggestion
* Add a new script `check-migration-dryrun` that allows to check if the migration can be safely run
```

**Comment by educhastenier** [comment]
:warning: You should separate this PR in two:
One part ready to merge for bonita 7.9.4
One part ready to merge for bonita 7.10

**Comment by baptistemesta** [comment]
My bad:
Depends on https://github.com/bonitasoft/bonita-doc/pull/819

**Pull request merged**